scss/titles.scss

```diff
@@ -3,9 +3,10 @@ $title-decoration-width: 70px;
 $text-big: 24px;
 
 .title {
-  font-size: 24px;
-    padding: 32px 0 32px 50px;
-    position: relative;
+  font-size: $text-big;
+  padding: 32px 0 32px 50px;
+  position: relative;
+  margin: 50px 0 15px 50px;
```

Comment:
this should be margin bottom and the 50px at the top and left should be done by adding a class `container` to the body and styling that class doing:
```
.container {
  padding: 50px;
}
```